Automattic/woocommerce-payments
# Fix styling on payment details page in mobile view.

Fixes https://github.com/Automattic/woocommerce-payments/issues/7230

#### Changes proposed in this Pull Request









- Fix the styling of the transaction status pill to be positioned at the top and not on the right of the transaction amount.

In my local, the pill does not overflow like shown in the screenshot of the [original issue](https://github.com/Automattic/woocommerce-payments/issues/7230).

Before:
![Screenshot 2024-11-25 at 12 53 53](https://github.com/user-attachments/assets/2174787f-952d-48ff-91db-f6398d308aa5)

After:
![Screenshot 2024-11-25 at 12 49 51](https://github.com/user-attachments/assets/2bb1795f-c187-475c-8ec8-7f58d2343dc6)

I have to re-arrange the elements. I group the transaction amount and currency and move the `PaymentStatusChip` and `DisputeStatusChip`.

I also added `className` prop for `PaymentStatusChip` and `DisputeStatusChip` so I can target them in CSS selector.

- Fix the styling of dispute notice link that overflow and breaks structure.

Before:
![Screenshot 2024-11-25 at 12 54 01](https://github.com/user-attachments/assets/cac8178a-da06-4a0d-8604-78ad71ab6b86)

After:
![Screenshot 2024-11-25 at 12 50 00](https://github.com/user-attachments/assets/bbe30ecc-c66c-45cd-be9b-542262e4403c)


#### Testing instructions





* Make a purchase using [dispute test cards](https://docs.stripe.com/testing#disputes).
* Open the dispute transaction. Payments > Transactions > click on the dispute transaction.

Make sure desktop view does not change due to these PR changes.

![Screenshot 2024-11-25 at 12 50 48](https://github.com/user-attachments/assets/ccd805ab-3fe7-4612-8b55-39f46001cb3b)

Make sure in mobile view, the status chips is located at the top.

![Screenshot 2024-11-25 at 12 49 51](https://github.com/user-attachments/assets/2bb1795f-c187-475c-8ec8-7f58d2343dc6)

* Open a charge transaction. Payments > Transactions > click on a charge transaction.

Make sure desktop view does not change due to these PR changes.

![Screenshot 2024-11-25 at 12 50 36](https://github.com/user-attachments/assets/f96fec08-b612-4d01-a003-f93ecd1d0af5)

Make sure in mobile view, the status chips is located at the top.

![Screenshot 2024-11-25 at 12 50 18](https://github.com/user-attachments/assets/0d616794-cb61-4c32-8564-d9bd0295a87e)


-------------------


**Post merge**
